ci/gpu/build.sh

```diff
@@ -78,11 +66,10 @@ conda list --show-channel-urls
 # BUILD - Build clx from source
 ################################################################################
 
-logger "Build clx..."
-"$WORKSPACE/build.sh" clean clx
-
-# FIX Added to deal with Anancoda SSL verification issues during conda builds
-conda config --set ssl_verify False
+logger "Build and install clx..."
+cd "${WORKSPACE}"
+gpuci_conda_retry build conda/recipes/clx
+gpuci_mamba_retry install clx
```

Comment:
shouldn't this `install` command use a local channel argument to ensure that it's installed the just-built version of `clx`? The build log shows it installing from `rapidsai-nightly`

Reply:
PR updated